apps/web-portal-client/lib/components/AppRouter.jsx

```diff
@@ -0,0 +1,27 @@
+import {useEffect} from 'react'
+import PropTypes from 'prop-types'
+import {useLocation} from 'react-router-dom'
+import {useAuth} from '../features/auth/hooks'
+import {appPersistentStorage} from '../utils'
```

Comment:
let's move appPersistentStorage storage to `AppContext`

Reply:
done